src/CMakeLists.txt

```diff
@@ -1608,6 +1608,7 @@ target_compile_definitions(
     $<$<COMPILE_LANGUAGE:CUDA>:__OFFLOAD_CUDA>
     $<$<BOOL:${CP2K_USE_HIP}>:__HIP_PLATFORM_AMD__
     __OFFLOAD_HIP>
+    $<$<BOOL:${CP2K_USE_UNIFIED_MEMORY}>:CP2K_USE_UNIFIED_MEMORY>
```

Comment:
Please pick a name that follows our convention for flag, e.g.  `__OFFLOAD_UNIFIED_MEMORY`.

Reply:
will do.